pkg/action/build-iso.go

```diff
@@ -136,14 +140,137 @@ func (b BuildISOAction) prepareISORoot(isoDir string, rootDir string, uefiDir st
 	}
 
 	b.cfg.Logger.Info("Creating squashfs...")
-	err = utils.CreateSquashFS(b.cfg.Runner, b.cfg.Logger, rootDir, filepath.Join(isoDir, constants.IsoRootFile), constants.GetDefaultSquashfsOptions())
+	//err = utils.CreateSquashFS(b.cfg.Runner, b.cfg.Logger, rootDir, filepath.Join(isoDir, constants.IsoRootFile), constants.GetDefaultSquashfsOptions())
+	err = CreateSquashFS(b.cfg, rootDir, filepath.Join(isoDir, constants.IsoRootFile))
```

Comment:
As much as I love to be proven wrong, IF we have to do this, let's do it gradually: first introduce it behind a feature flag, and promote it when it's mature enough.

 I don't want us to start chasing bugs down the line now that we have this component stable!